README.md

```diff
@@ -125,7 +122,7 @@ For reference here's how a full configuration could look:
    - mobile_application_version_file_path: 'path/to/application.apk'
    - polling_timeout: 4200000
    - public_ids: 'abc-d3f-ghi, jkl-mn0-pqr'
-   - site: 'datadoghq.com'
+   - site: {{< region-param key="dd_site" code="true" >}}
```

Comment:
Could we avoid using `{{ }}` markup in publicly available markdown documents? It'll be displayed as is. Instead I believe we should use `<!-- -->` markup, like [on the datadog-ci synthetics readme](https://github.com/DataDog/datadog-ci/blob/bff12e212215133d0eb59ae1814d052a2ee886c2/src/commands/synthetics/README.md?plain=1#L68).
However, because this line is within a code block, I don't think the markup would work, so I'd rather not have anything here, and keep the simple `datadoghq.com`.
```suggestion
   - site: 'datadoghq.com'
```